.travis.yml

```diff
@@ -31,3 +30,8 @@ script:
         health="$(docker inspect --format='{{.State.Health.Status}}' local-test-container)"
         test "${health}" = "healthy"
     )
+
+after_failure:
```

Comment:
Is there no `finally`?

Jenkins doesn't support those clauses.

Reply:
`script` is bash, there is no `finally` (and manually doing finally-like constructs is verbose)

for internal infra like jenkins other things would need to be changed too, like containers cleanup